libs/domain/picking/src/components/picking-product-card/picking-product-card.component.ts

```diff
@@ -0,0 +1,184 @@
+
+  @state() isCorrectNumberOfPickedProvided = true;
+  @state() currentNumberOfPicked?: number;
+  @state() pickedDataEvent?: ProductItemPickedEvent;
+
+  protected summaryInfo: SummaryInfo | undefined;
+
+  protected onSubmit(e: SubmitEvent): void {
+    e.preventDefault();
+
+    this.currentNumberOfPicked =
+      this.currentNumberOfPicked ?? this.productItem?.numberOfPicked;
+
+    if (this.productItem?.product.id) {
+      this.dispatchPickingEvents(EVENT_SUBMIT, {
+        productId: this.productItem.product.id,
+        numberOfPicked: this.currentNumberOfPicked,
+      });
+    }
+  }
+
+  protected editProductPicking(): void {
+    this.dispatchPickingEvents(EVENT_EDIT, {
+      productId: this.productItem?.product.id,
+    } as ProductItemPickedEvent);
+  }
+
+  protected onChangeQuantity({ detail: { quantity } }: CustomEvent): void {
+    this.currentNumberOfPicked = quantity;
+  }
+
+  protected getSummaryInfo(): SummaryInfo {
+    if (!this.productItem) {
+      return {
+        main: '',
+      };
+    }
+
+    if (this.status === ItemsFilters.Picked) {
+      const template = `${this.productItem.numberOfPicked}/${this.productItem.quantity}`;
+
+      if (this.productItem.numberOfPicked < this.productItem.quantity) {
+        return {
+          main: `${template} ${i18n('picking.product-card.items-picked')}`,
+        };
+      }
+
+      return {
+        main: template,
+        additional: `${i18n('picking.product-card.all-items-picked')}`,
+      };
+    } else if (this.status === ItemsFilters.NotFound) {
+      const template = `${this.productItem.numberOfNotPicked}/${this.productItem.quantity}`;
+
+      if (this.productItem.numberOfNotPicked < this.productItem.quantity) {
+        return {
+          main: `${template} ${i18n('picking.product-card.items-not-found')}`,
+        };
+      }
+
+      return {
+        main: template,
+        additional: `${i18n('picking.product-card.no-items-found')}`,
+      };
+    }
+
+    return {
+      main: '',
+    };
+  }
+
+  protected dispatchPickingEvents(
+    event: string,
+    productItem?: ProductItemPickedEvent
+  ): void {
+    this.dispatchEvent(
+      new CustomEvent(event, {
+        bubbles: true,
+        composed: true,
+        detail: productItem,
+      })
+    );
+  }
+
+  protected override render(): TemplateResult {
+    return html`${this.renderPickingProduct()}`;
+  }
+
+  protected renderPickingProduct(): TemplateResult {
+    if (this.productItem && this.currentNumberOfPicked) {
+      this.isCorrectNumberOfPickedProvided =
+        0 <= this.currentNumberOfPicked &&
+        this.currentNumberOfPicked <= this.productItem.quantity;
+    }
+
+    return html`
+      ${when(
+        this.productItem,
+        () => html`
+          <oryx-card>
+            <oryx-heading slot="heading" class="title">
+              ${this.productItem?.orderItem?.name}
+            </oryx-heading>
+            <div class="subtitle">${this.productItem?.orderItem.sku}</div>
```

Comment:
we can use `oryx-heading` here and avoid using extra styles

Reply:
There is `h4` tag with the same styles.
You can find typography styles here:
![image](https://user-images.githubusercontent.com/43376934/228216341-1624206c-56cd-400d-ab0f-105d25931a95.png)

Reply:
Looks that you should update your branch with development

Reply:
yup, It was updated with this changes from 7 days ago, I will use this instead